e2e/acl/acl_token_test.go

```diff
@@ -129,43 +129,43 @@ func testACLTokenExpiration(t *testing.T) {
 	defaultNSQueryMeta.AuthToken = tokenQuickExpiryCreateResp.SecretID
 
 	jobListResp, _, err = nomadClient.Jobs().List(&defaultNSQueryMeta)
-	require.ErrorContains(t, err, "ACL token expired")
-	require.Nil(t, jobListResp)
+	must.ErrorContains(t, err, "Permission denied")
```

Comment:
just flagging this as the functional test fix amongst the require->must conversions